Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
 name = "hubcaps"
-version = "0.4.10"
+version = "0.5.0"
 authors = ["softprops <d.tangren@gmail.com>"]
 description = "Rust interface for Github"
-documentation = "https://softprops.github.io/hubcaps"
+documentation = "https://docs.rs/hubcaps/"
```

Comment:
this makes crates.io point at release documentation which is more inline with what a crates.io user will want as crates.io is a release host